index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Shoko WEB UI</title>
+      <link rel="shortcut icon" href="favicon.ico" />
+  </head>
+  <body>
+    <link rel="stylesheet" type="text/css" href="/api/v3/WebUI/Theme.css" /> <!-- This is inside body because it needs to be loaded after the style tags in dev server -->
```

Comment:
This should only be in the body for the dev server though.

Reply:
I'm not quite sure how Theme.css works, to be completely honest, I was testing on an old build of shoko server

Reply:
The server just provides an auto-generated css file with the theme declarations. The theme declarations are just like the `.theme-shoko-gray` declaration (now) found under `src/css/theme-gray.css` (which should be renamed to `src/css/theme-shoko-gray.css` imo), and applying a theme is as simple as applying the theme class name to an element, be it our root element or any other element on the page.